### ISSUE TITLE
State not announced on iOS 11 w/VO

### ISSUE THREAD

**Issue by scottaohara** [scottaohara/aria-switch-button#3]
Confirmed that checked/unchecked state not announced on iOS 11 with VoiceOver enabled.  

iOS VO announces the item as a checkbox, and not a `switch`, as it is correctly announced on VO + Desktop.

This could possibly be a bug due to iOS Safari not understanding the `role="switch"`.  

Investigate fallbacks for appropriate announcements for browsers/AT combos that don't support `switch`
